publication-commons/src/main/java/no/unit/nva/publication/service/impl/ResourceService.java

```diff
@@ -200,7 +200,7 @@ public void updateOwner(SortableIdentifier identifier, UserInstance oldOwner, Us
     }
     
     public Publication updatePublication(Publication resourceUpdate) {
-        return updateResourceService.updatePublication(resourceUpdate);
+        return updateResourceService.updatePublicationButDoNotChangeStatus(resourceUpdate);
```

Comment:
update publication used to allow anything to be updated. This was kind of technical debt that we should fix at this point because we are changing more than one fields when we are publishing a publication.